.gitignore

```diff
@@ -7,3 +7,4 @@
 *.swp
 *.swo
 playground/*
+.idea
```

Comment:
This should go into your global `.gitignore`

Reply:
@Plytas is right! @aifodu Can you remove this entry?

Reply:
Yes I can and I have, but I notice another request has already been made without any conflict to resolve this.